ncl/ncl_entries/climatology_functions.ipynb

```diff
@@ -0,0 +1,313 @@
+{
```

Comment:
<p>Also here, the width of the plot is cutting off the title values</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/geocat-applications/pull/80/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>